src/koaApi.coffee

```diff
@@ -77,10 +79,27 @@ exports.setupApp = (done) ->
   app.use route.get '/visualizer/events/:receivedTime', visualizer.getLatestEvents
   app.use route.get '/visualizer/sync', visualizer.sync
 
-  app.use route.get '/metrics', metrics.getGlobalLoadTimeMetrics
-  app.use route.get '/metrics/status', metrics.getGlobalStatusMetrics
-  app.use route.get '/metrics/:type/:channelId', metrics.getChannelMetrics
-  
+# -- Old metrics Routes Purposely commented out for Backward Compatibility
+  app.use route.get '/mongometrics', metrics.getGlobalLoadTimeMetrics
+  app.use route.get '/mongometrics/status', metrics.getGlobalStatusMetrics
+  app.use route.get '/mongometrics/:type/:channelId', metrics.getChannelMetrics
```

Comment:
I am not sure how to do this locally. as far as I know that config can only be set globally for all tests. Unless I put overide routes where a user can specifically request metrics from mongo, or from statsd